include/cobs/stream.h

```diff
@@ -4,7 +4,11 @@
 #define COBS_STREAM_H_
 
 #include <stdbool.h>
+#if ((KERNEL_VERSION_MAJOR) < 3) || (((KERNEL_VERSION_MAJOR == 3)) && ((KERNEL_VERSION_MINOR) < 1))
```

Comment:
which version are you targeting / testing with? 
The only supported version that would fit this check is Zephyr 2.7.5. Everything else is EOL.
https://docs.zephyrproject.org/latest/releases/index.html

Reply:
The problem exhibits on Conhub project which uses Zephyr v 2.2.1